test/web-auth/captcha.test.js

```diff
@@ -544,17 +528,22 @@ describe('captcha rendering', function () {
         expect(input.value).to.not.equal('');
       });
 
-      it('should run arkose when calling runArkose()', function () {
-        c.runArkose();
+      it('should return true for requiresTrigger()', function () {
```

Comment:
Can we add at least 1 counter-assertion, that it returns `false` for other providers?